NevisExampleApp/Utility/Validation/AccountValidator.swift

```diff
@@ -24,7 +24,7 @@ extension AccountValidator {
 		}
 
 		var accounts = [any Account]()
-		supportedAuthenticators.forEach { authenticator in
+		for authenticator in supportedAuthenticators {
```

Comment:
I think the SwiftFormat rule is missing here not to rewrite the forEach

Reply:
Fixed.